src/inc/clrnt.h

```diff
@@ -789,6 +789,8 @@ typedef struct _DYNAMIC_FUNCTION_TABLE {
 //
 #ifdef _TARGET_AMD64_
 
+#define COMPACT_RUNTIME_FUNCTION        0
```

Comment:
You do not need the EndAddress on x86. Having it for x86 is just wasting space - encoding the length in xdata is more compact. EndAddress on Win x64 was a mistake. It would not be there if we can go back in time. 

It has nothing to do with instruction with. E.g. ARM32 has variable instruction width as well (the instruction is either 2 or 4 bytes long) and it is able to run without EndAddress just fine.

The multiplier has to do with instruction size granularity. For x86, the granularity is 1 byte so you should make the multiplier 1.